src/main/resources/static/bidder-params/taboola.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Taboola Adapter Params",
+  "description": "A schema which validates params accepted by the Taboola adapter",
+
```

Comment:
Please, remove this empty line. `cmd+option+L` or `ctrl+alt+L`, if you work in idea.